scripts/ci/tools/free-space.py

```diff
@@ -0,0 +1,43 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""free-space.py for cean environment before to start CI"""
+
+import subprocess
+
+exit_code = subprocess.call('./../libraries/_script_init.sh', shell=True)
+print(exit_code)
```

Comment:
I guess we don't need it because we don't use any variables, only one `docker_v` function which we can rewrite into Python.

Reply:
Yep. Agree,

Reply:
@mik-laj do you mean that it is not necessary to call the script: _script_init.sh or it is not necessary to print the response of that call.

Reply:
I got is mik-laj, I am considering just this:
`subprocess.call('./../libraries/_script_init.sh', shell=True)`

Reply:
`./../libraries/_script_init.sh` initializes environment variables and bash functions, but you can't use them in Python, so it doesn't make sense to load this file at all.

Reply:
Got it! I removed!